tasks/add_odl_deb_repo.yml

```diff
@@ -0,0 +1,18 @@
+- name: Add the repository key to apt
+  apt_key:
+    url={{ deb_key }}
+    state=present
+
+- name: Add ODL deb repo
+  apt_repository:
+    repo={{ deb_repo_url }}
+    state=present
+    filename='opendaylight'
```

Comment:
Is this the normal file naming convention? The repo files end in .repo for yum, which is why I noticed.